db/seeds.rb

```diff
@@ -4,8 +4,7 @@
 user = User.find_or_initialize_by(:email => ENV['SEED_EMAIL'] || "admin@example.com")
 
 if user.persisted?
-  puts "User with email '#{user.email}' already exists, not seeding."
-  exit
+  raise "User with email '#{user.email}' already exists, not seeding."
```

Comment:
My apologies, I didn't realize that seeds are expected to get run over and over on the same db. Leaving it as it it was makes total sense. Maybe warrants a test like `it "allows seeds to be run multiple times without halting execution"`?